src/libman/Commands/CacheListCommand.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.CommandLineUtils;
+using Microsoft.Web.LibraryManager.Contracts;
+
+namespace Microsoft.Web.LibraryManager.Tools.Commands
+{
+    /// <summary>
+    /// Defines the cache list command.
+    /// Allows the user to display the contents of the libman cache.
+    /// </summary>
+    internal class CacheListCommand : BaseCommand
```

Comment:
Do we need this?

Reply:
The command itself? 
How else would the user know what is installed in the cache? 
Or is the question that do we need to allow the user to know what is in the cache?

Reply:
The second.  I sent an email about this... I probably missed the discussion on this. Is this to show the catalog (list of libraries available in a given provider) or to show the full libraries you have on cache?